Automattic/WP-Job-Manager
# Add ability to filter by `post_status` in `[jobs]`; persist `job_types`

Fixes #837 
Fixes #788 

#### Changes proposed in this Pull Request:

* `[jobs]` shortcode: Fixes issue with persisting job_types when `show_filters` is 0.
* `[jobs]` shortcode: Adds ability to filter by `post_status`

#### Testing instructions:

* With a `per_page="1"`, verify `post_status` and `job_types` persist (e.g. `[jobs job_types="cool" post_status="publish,expired" per_page="1" show_filters="false"]`)


#### Proposed changelog entry for your changes:
* Enhancement: Adds the ability to filter `[jobs]` by `post_status` with new shortcode attribute. (#788)
* Fix: Persist `job_types` attribute in `[jobs]` shortcode between pages. (#837)
